File: src/test/java/ParserTest.java
import org.junit.Test;

import java.util.List;

/**
 * Author: stk
 * Date: 18/4/3
 */
public class ParserTest {
    @Test
    public void parseProject() {
        Parser parser = new Parser();
        List<String> result = parser.parseProject("C:\\Users\\stk\\Downloads\\transport");
        result.forEach(System.out::println);
    }
}

File: src/test/scala/ParserTest.scala
import java.io.File

import org.junit.Test

/**
  * Author: stk
  * Date: 18/4/6
  */
class ParserTest {
  val path = "C:\\Users\\stk\\Downloads\\test-data\\projects\\26-netty\\buffer\\src\\main\\java\\io\\netty\\buffer\\AbstractByteBuf.blame"

  @Test
  def parseFile(): Unit = {
    val result = FileParser.parseFile(new File(path))
    if (result != null) result.foreach(println)
  }
}
